pkgs/os-specific/darwin/apple-source-releases/file_cmds/package.nix

```diff
@@ -1,11 +1,9 @@
 {
   lib,
   apple-sdk,
-  apple-sdk_11,
```

Comment:
file_cmds is failing to build for me due to being unable to find the `sha.h` header.